DIPs/001/README.md

```diff
@@ -0,0 +1,20 @@
+# Minimum Viable Governance
+This document describes the Minimum Viable Governance of Social Dist0rtion Protocol.
+
+The funds of the DAO are managed by the multi signature wallet deployed in the Main Ethereum Network at the address `0x527621278422FFf45A66f7086bAeAC770CF12b69`
+
+The multi signature wallet defines a *set of owners* and a *number of required confirmations* to execute a transaction.
+
+Anyone can make a **proposal** by opening an issue in any repository of the Social Dist0rtion Protocol organization on GitHub.
+
+The proposal must contain:
+- The *problem* being addressed.
+- The *solution* proposed.
+- An *estimated effort*.
+- The *payment conditions*.
+
+The author of the proposal has the responsibility to gauge interest around their work, but it is not an obligation of the DAO to take their work into consideration.
+
+Being `n` the number of confirmations required to execute a transaction:
+- A proposal is **accepted** when at least `n` owners vote for it.
```

Comment:
how is "voting" happening?

Reply:
The voting to accept a proposal can be done offchain, to avoid calling a contract on-chain (this would be slow and cost money).

The voting can happen in the comments section of the issue. There should be a way to verify that the comment is coming from one of the owners. In this case we might need to implement a "sign and verify" page where a user can verify that the signatures are coming from the owners.

I'd love to have a simpler solution, if you have anything in mind please share :pray:

Reply:
comment voting i think is great.